.NET/Microsoft.Recognizers.Text/Models/ResolutionKey.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Microsoft.Recognizers.Text
+{
+    public static class ResolutionKey
+    {
+        public const string ValueSetKey = "values";
```

Comment:
If the class is already ResolutionKey, no need to repeat the suffix Key for every key. :)